.github/README.md

```diff
@@ -779,8 +779,16 @@ Click the button below, to deploy to Vercel 👇
 
 ### Deploying - Option #3: Docker
 
+Pull Image: `docker pull lissy93/web-check`
+
 Run `docker run -p 3000:3000 lissy93/web-check`, then open [`localhost:3000`](http://localhost:3000)
 
+or:
+
+Run `docker -d --name webcheck -p 3000:3000 --restart=always lissy93/web-check`, then open [`localhost:3000`](http://localhost:3000)
```

Comment:
+1 to this. Or maybe a `docker-compose.yml` would be more suitable, as that's what most people are going to use to deploy, and it gets a bit hard to read with many docker flags.

Reply:
Sorry for the delay. I haven't had time outside of work to address this. Working it now.